### PR TITLE
Copies Enterprise Helm Chart

### PR DESCRIPTION
**Pull request by rodrigovalin**
# Summary

Copies the Enterprise Chart from the private repository in order to have everything up-to-date in this repository. We are working on releasing new versions as part of our release process.

### CODE REVIEW

**Comment by kwill4026** [comment]
@rodrigovalin Hello Rodrigo. Are you all working on getting helm charts for the database portions and ops manage portion of the MongoDB Kubernetes Enteprise Operator? I know currnetly its only available for the Operator to be deployed using helm charts.

**Comment by rodrigovalin** [comment]
@kwill4026 Hi!

Yes we are; we want to have the database and Ops Manager resources managed by Helm Charts as well. Leave your comments here; all feedback is welcome.

**Comment by kwill4026** [comment]
> @kwill4026 Hi!
> 
> Yes we are; we want to have the database and Ops Manager resources managed by Helm Charts as well. Leave your comments here; all feedback is welcome.

Thats great to know you all are working on it. Any estimated time of completion on this?

I've been following the chart helpers from here https://github.com/mongodb/helm-charts/blob/main/helpers/MongoDB-tls.sh and trying to incorporate into my git clone of the MongoDB Enterprise Kubernetes Operator. So far so good BUT I need to figure out how to get the secrets for the certs and configmap of the CA to be included in the helm chart instead of having a script that creates them and then calls the helm install of the chart. 
I would love to see where you guys are at now on this though as my way is not how I would like for it to be moving forward.

File: charts/enterprise-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Prerequisites
+
+The installation of this Chart does not have prerequisites. However, in order to
+create a Mongo Database in your Kubernetes cluster, you'll need a [Cloud
+Manager](https://cloud.mongodb.com) account.
+
+## Installing Enterprise Operator
+
+You can install the MongoDB Community Operator easily with:
```
Comment (jasonmimick) [inline charts/enterprise-operator/README.md:14]: check this please

**Review by jasonmimick — CHANGES_REQUESTED** [review]

**Comment by rodrigovalin** [comment]
@jasonmimick ready for another review.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ This repository contains Helm Charts for different MongoDB products.
 
 | Charts                                            | Description                                                               |
 | ------------------------------------------------- | ------------------------------------------------------------------------- |
-| [atlas-operator](charts/atlas-operator)           | MongoDB Atlas Operator Helm Chart. [_Start Here!_](charts/atlas-operator) |
+| [enterprise-operator](charts/enterprise-operator) | MongoDB Enterprise Kubernetes Operator Helm Chart.                        |
```
Comment (jasonmimick) [inline README.md:9]: How about we put ent at bottom - then it goes from "simpler" to "less simpler"

File: charts/enterprise-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# MongoDB Enterprise Kubernetes Operator Helm Chart
+
+A Helm Chart for installing and upgrading the [MongoDB Enterprise
+Kubernetes Operator](https://github.com/mongodb/mongodb-enterprise-kubernetes).
+
+## Prerequisites
+
+The installation of this Chart does not have prerequisites. However, in order to
+create a Mongo Database in your Kubernetes cluster, you'll need a [Cloud
+Manager](https://cloud.mongodb.com) account.
```
Comment (jasonmimick) [inline charts/enterprise-operator/README.md:10]: Or Ops Manager installed - can we add a link here

File: charts/enterprise-operator/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Please visit "[Configure Kubernetes for Deploying MongoDB
+Resource](https://docs.cloudmanager.mongodb.com/tutorial/nav/k8s-config-for-mdb-resource/)"
+in order to create new `Secret`s and `ConfigMap`s with credentials for your
+Cloud Manager project.
+
+You will create a `ConfigMap` with name `my-project`, and a `Secret` with name
+`my-credentials`.
+
+## Deploying a MongoDB Replica Set
```
Comment (jasonmimick) [inline charts/enterprise-operator/README.md:46]: We need to add the chart for an enterprise operator replica set (with enterprise backing db)
Then also we should have some comments on how to use community operator with the enterprise dbs.

File: charts/enterprise-operator/crds/webhook-cluster-role.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (jasonmimick) [inline charts/enterprise-operator/crds/webhook-cluster-role.yaml:1]: This is a newer way to install this; is there any notes/docs we can add?
How does this effect watch namespace?

File: charts/enterprise-operator/values.yaml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  operator: quay.io/mongodb
+  database: quay.io/mongodb
+  initDatabase: quay.io/mongodb
+  initOpsManager: quay.io/mongodb
+  opsManager: quay.io/mongodb
+  initAppDb: quay.io/mongodb
+  appDb: quay.io/mongodb
+  agent: quay.io/mongodb
+
+multiCluster:
```
Comment (jasonmimick) [inline charts/enterprise-operator/values.yaml:79]: 👍🏻 @irajdeep @chatton fyi

**Review by jasonmimick — APPROVED** [review]

File: charts/enterprise-operator/README.md (hunk 4 above)
Comment (rodrigovalin) [reply charts/enterprise-operator/README.md:46]: Yes, we have time allocated for that and will do as a follow-up to this patch.

File: charts/enterprise-operator/crds/webhook-cluster-role.yaml (hunk 5 above)
Comment (rodrigovalin) [reply charts/enterprise-operator/crds/webhook-cluster-role.yaml:1]: This is not a new way. These are the exact same resources we have in Enterprise in the [public repository](https://github.com/mongodb/mongodb-enterprise-kubernetes/blob/master/helm_chart/crds/webhook-cluster-role.yaml).

Now, I don't think this belongs in `crds/` directory, but will leave it there for now. We can revisit this later.

**Pull request merged**